covid_tracing/src/main.rs

```diff
@@ -50,8 +56,9 @@ fn main() {
                     .into_iter()
                     .flat_map(|(pid_a, pid_b, t)| vec![(pid_a, (pid_b, t)), (pid_b, (pid_a, t))]);
 
+                let mut join_state = context.state_ref(state_handle).unwrap().borrow_mut();
```

Comment:
If these `unwrap`s are guaranteed to be safe, we should do it automatically, I think? And maybe even `borrow_mut()` it for you too. Put another way, I think we should make sure that user code can be written without any `unwrap`s for basic stuff.

Reply:
This is true right now. My logic was that in the future it may be possible to drop state. However I guess with the Rc idea than we won't hit this situation so Ill put the unwrap back inside